PhaseAndSave3DLineNarrowV1.ipf

```diff
@@ -583,16 +583,19 @@ Function Open_File_source(ctrlName): ButtonControl
 		if ((Abs(source_wave_real[ipntnum])<1e-9)&&(Abs(source_wave_imag[ipntnum])<1e-9))
 			InZeroBool=1
 			iAtEndOfFirstZeroBurst=ipntnum //use this to hold the current Zero value
+			break
 		endif
-		//debugging...worked...no longer needed//Print "ipntnum=", ipntnum,  "Re=", Abs(source_wave_real[ipntnum]), "Im=", Abs(source_wave_imag[ipntnum]), "InZeroBool=", InZeroBool
 		ipntnum+=1
-	While ((((Abs(source_wave_real[ipntnum])<1e-9)&&(Abs(source_wave_imag[ipntnum])<1e-9)) || (InZeroBool==0)) && (ipntnum<source_numpnts))
```

Comment:
All this stuff in the While condition is obviated by adding a break statement to the if statement